src/fr/reaperscans/src/eu/kanade/tachiyomi/extension/fr/reaperscans/ReaperScans.kt

```diff
@@ -1,13 +1,33 @@
 package eu.kanade.tachiyomi.extension.fr.reaperscans
 
 import eu.kanade.tachiyomi.multisrc.keyoapp.Keyoapp
+import eu.kanade.tachiyomi.source.model.SManga
+import org.jsoup.nodes.Document
 
 class ReaperScans : Keyoapp(
     "Reaper Scans",
     "https://reaper-scans.fr",
     "fr",
 ) {
 
+    override fun mangaDetailsParse(document: Document): SManga =
+        super.mangaDetailsParse(document).apply {
+            // Fix a typo in the class name
+            if (description.isNullOrBlank() == true) {
+                description = document.selectFirst("div.grid > div.overflow-hiddfen > p")?.text()
+            }
+
+            // Check that text is "Statut"
+            val statusSpan =
+                document.select("div:has(> img) > span").firstOrNull { it.text() == "Status" }
+            status =
+                statusSpan?.parent()?.parent()?.selectXpath("div[2]")?.firstOrNull().parseStatus()
```

Comment:
```css
div:has(> div > img):has(> div > span:containsOwn(Status)) > div:last-child
```
or
```css
div:has(> span:containsOwn(Status)) + div
```
or
```css
div:has(> img[src*=status]) + div
```

Reply:
I will do the last one, as this is vanilla w3c html5